src/app/data-planes/views/DataPlaneListView.vue

```diff
@@ -18,16 +18,27 @@
         </template>
         <KCard>
           <template #body>
-            <DataPlaneList
+            <AppCollection
               data-testid="data-plane-collection"
               class="data-plane-collection"
+              :empty-state-title="t('common.emptyState.title')"
+              :empty-state-message="t('common.emptyState.message', { type: 'Data plane proxies' })"
+              :headers="[
+                { label: 'Name', key: 'name' },
+                { label: 'Service', key: 'service' },
+                { label: 'Protocol', key: 'protocol' },
+                isMultiZoneMode ? { label: 'Zone', key: 'zone' } : undefined,
+                { label: 'Last Updated', key: 'lastUpdated' },
+                { label: 'Kuma DP version', key: 'dpVersion' },
+                { label: 'Status', key: 'status' },
+                { label: 'Actions', key: 'actions', hideLabel: true },
+              ].filter(Boolean)"
```

Comment:
**Suggestion**: Can you please use the `notEmpty` utility here? I’m changing `headers` in `AppCollection` ([here](https://github.com/kumahq/kuma-gui/pull/1259)) to have the right types which would make this error so doing this now avoids that becoming a conflict.

```suggestion
              ].filter(notEmpty)"
```